Ex#1/ecasa_inv/src/main/java/com/avangenio/ecasa_inv/controller/InventoryController.java

```diff
@@ -0,0 +1,90 @@
+package com.avangenio.ecasa_inv.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.avangenio.ecasa_inv.detail.InventoryDetail;
+import com.avangenio.ecasa_inv.detail.ProductDetail;
+import com.avangenio.ecasa_inv.detail.SectionDetail;
+import com.avangenio.ecasa_inv.service.InventoryService;
+import com.avangenio.ecasa_inv.service.ProductService;
+import com.avangenio.ecasa_inv.service.SectionService;
+
+@RestController
+@RequestMapping("/ecasa/inventories")
+public class InventoryController {
+	
+	@Autowired
+	private InventoryService inventoryService;
+	
+	@Autowired
+	private ProductService productService;
+	@Autowired
+	private SectionService sectionService;
+	
+	@GetMapping
+	public List<InventoryDetail> getAll() {
+		return inventoryService.findAll();
+	}
+	
+	@PostMapping("{sectionId}/product/{productId}")

```

Comment:
Using path variables for specifying entity data instead of post payload is unorthodox.